src/ast.js

```diff
@@ -2673,12 +2673,27 @@ function parsestrplus (c, n) {
     }
 }
 
+const invalidSyntax = /_[eE]|[eE]_|\._|j_/;
+const invalidDecimalLiteral = /_\.|[+-]_|^0_\D|_j/;
```

Comment:
what am i missing here? - are these two regex tests not part of the slow path too? They're checking that underscores don't immediately proceed/follow something?

Reply:
Oops lol, I missed that. Sorry, I've been tired haha